### ISSUE TITLE
NConcern 4.0.3 for .NET 4.0

### ISSUE THREAD

**Issue by alfhv** [Virtuoze/NConcern#10]
NConcern nuget package 4.0.3 says it is targeted for .NET 4.0+ but when trying to build on a .NET 4.0 project I got error. 
Inspecting assembly packages\NConcern.4.0.3\lib\NConcern.dll it says:

[assembly: TargetFramework(".NETFramework,Version=v4.5.2", FrameworkDisplayName = ".NET Framework 4.5.2")]

Could be possible to have a 4.0.3 version targeting .NET 4.0 ?
thanks,

**Comment by Virtuoze** [comment]
You're right, I will take this consideration to enable async injection only for .net 4.5.2+
Thank you for reporting it. NConcern will be downgrade to 4.0 and async support will only appear in Puresharp (NConcern successor).
